apis/account/v1alpha1/subscription_types.go

```diff
@@ -27,6 +28,9 @@ type SubscriptionParameters struct {
 	// PlanName to subscribe to, empty plannames are shown as "default" in cockpit, use "" instead
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="planName can't be updated once set"
 	PlanName string `json:"planName"`
+	// Subscription parameters allows you to add additional parameters
+	// +kubebuilder:validation:Optional
+	SubscriptionParameters runtime.RawExtension `json:"subscriptionParameters"`
```

Comment:
```suggestion
	SubscriptionParameters runtime.RawExtension `json:"parameters"`
```

maybe would fit better?

Reply:
are

Reply:
Thank you, @maximilianbraun, for the comment.

The change is implemented in https://github.com/SAP/crossplane-provider-btp/pull/25/commits/145114d49d978f3c653de67c7ecefac2166de268